src/main/java/net/minecraftforge/common/conditions/ConditionHelper.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.conditions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import com.mojang.logging.LogUtils;
+
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraftforge.common.conditions.ICondition.IContext;
+
+/**
+ * 
+ * Holds code relating to the de/serialization and processing of {@link ICondition}s
+ */
+public class ConditionHelper {
+
+	private static final Logger LOGGER = LogUtils.getLogger();
+	
+	public static final Map<ResourceLocation, IConditionSerializer<?>> conditions = new HashMap<>();
```

Comment:
Why is this field `public`?

Reply:
Not sure.  It was public before, but I don't see a compelling reason for it to need to be public.